lib/components/formik/Input.js

```diff
@@ -7,7 +7,7 @@ export default function FormikInput({ name, ...rest }) {
   return (
     <Field name={name}>
       {({ field, meta }) => (
-        <Input error={meta.touched && meta.error} {...rest} {...field} />
+        <Input {...field} {...rest} error={meta.touched && meta.error} />
```

Comment:
```suggestion
        const errorMessage = errors[field.name] || rest.error;
        return (
          <Input {...field} {...rest} error={meta.touched && errorMessage} />
        );
```
It is like this now in NeetoHelp right?